ServerCore/Components/Layout/MainLayout.razor

```diff
@@ -1,15 +1,7 @@
 ﻿@inherits LayoutComponentBase
 
 <div class="page">
-@*     <div class="sidebar">
```

Comment:
Just want to confirm that this is removing some comments left over from the .net 8 conversion, and not a functional change in any way.